waspc/src/Generator/FileDraft/WriteableMonad.hs

```diff
@@ -48,7 +51,14 @@ class (Monad m) => WriteableMonad m where
 
 instance WriteableMonad IO where
     createDirectoryIfMissing = System.Directory.createDirectoryIfMissing
-    copyFile = System.Directory.copyFile
+    copyFile src dst = do
```

Comment:
Put a TODO saying that we should rename this function, otherwise people expect it to crash if file does not exist.